### DISCUSSION TITLE
Add more fortification objects in logistic box

### DISCUSSION THREAD

**Discussion by mattia194** [Vdauphin/HeartsAndMinds#1666] [Question and answer]
I wanted to add more objects in the logistic point, specifically from the "boxloader" mod, which adds barriers without sand, which makes you able to put a bunch of them in a truck to make a fob faster. I did find the classnames in the mission.sqf. But, every time I add then, it gets me some CBA errors, anyone knows how to do this?

**Comment by Vdauphin** [comment]
Hello,
Add it to the array https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L482
Cheers
